recipes/napari-cilia-beating-frequency/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "napari-cilia-beating-frequency" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-cilia-beating-frequency-{{ version }}.tar.gz
+  sha256: 0f822cb352c5b3217919b9664aea004a8e948bf6db68bef6f3318e84dc86baa6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - magicgui
+    - napari
+    - numpy >=1.20
+    - python >=3.8
+    - qtpy
+    - scipy
+
+test:
+  imports:
+    - napari_cilia_beating_frequency
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/napari-cilia-beating-frequency/
```

Comment:
This is 404-ing, so maybe the plugin was deleted?

Reply:
Ooops, I meant the "homepage" link on there: https://github.com/emilmelnikov/napari-cilia-beating-frequency

Reply:
Pinging @emilmelnikov :)

Reply:
Hi! Sorry for stalling this issue, I didn't have access to a laptop recently :)

As discussed earlier in https://github.com/conda-forge/staged-recipes/pull/19674#issuecomment-1220107061, I removed the `Framework :: napari` classifier, but forgot to push and upload the new version, sorry for that.

This plugin is actually a very early prototype, and the work on it stopped a while ago, so I don't think the wider community will benefit from it, at least in it's current state.

Reply:
Thanks for the heads up!